engine/pmem_allocator/free_list.hpp

```diff
@@ -190,14 +190,17 @@ class Freelist {
           last_used_entry_ts(0) {}
 
     // Entry size stored in block unit
-    std::vector<std::vector<SpaceEntry>> active_entries;
+    FixedVector<std::vector<SpaceEntry>> active_entries;
```

Comment:
Why not use std::array or std::vector initialized to certain size?

Reply:
we need variable size